assignments/callbacks.js

```diff
@@ -22,31 +22,88 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+//function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+//}
+
+function getLength(arr, cb) {
+  return cb(arr.length);
 }
 
-function last(arr, cb) {
+getLength(items, function(length){
+  console.log(length);
+})
+
+
+// function last(arr, cb) {
   // last passes the last item of the array into the callback.
+// }
+
+function last(arr, cb) {
+  return cb(arr[arr.length-1]);
 }
 
-function sumNums(x, y, cb) {
+last(items, function(last){
+  console.log(last);
+})
+
+
+//function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+//}
+
+function sumNumbs(x, y, cb){
+  return cb(x+y);
 }
 
-function multiplyNums(x, y, cb) {
+sumNumbs(1, 2, function(sum){
+  console.log(sum);
+})
+
+
+//function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+//}
+
+function multipleNumbs(x, y, cb) {
+  return cb(x*y);
 }
 
-function contains(item, list, cb) {
+multipleNumbs(2, 3, function(product){
+  console.log(product);
+})
+
+
+//function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+//}
+
+function contains(item, list, cb){
+  
+  if (list.indexOf(item) === -1){
```

Comment:
So in case you hadn't already read about this yourself by now the reason this works is because indexOf returns -1 when the argument isn't in the array it's invoked on.  Probably use includes() or Set, those seem like better tools for this to me.  Still this is good.